alcs-frontend/src/app/features/home/assigned/assigned.component.scss

```diff
@@ -26,9 +26,19 @@
   }
 
   tr:hover {
-    box-shadow: 0 0 0 1px rgba($primary-color, .9);
+    box-shadow: 0 0 0 1px rgba($primary-color, 0.9);
```

Comment:
Was this auto-format or your change? I've been writing them as .2 just wondering if I should always include the 0.

Reply:
I do not think that is a hard requirement, but yes, it is automatic formatting